buf.gen.yaml

```diff
@@ -2,4 +2,4 @@ version: v1
 plugins:
   - plugin: go
     out: .
-    opt: module=github.com/mercari/grpc-federation
```

Comment:
We cannot specify the module if the source contains multiple modules, so I updated it to `paths=source_relative`